packages/logging-winston/system-test/logging-winston.js

```diff
@@ -68,34 +71,48 @@ describe('LoggingWinston', function() {
             util.inspect(testTimestamp)
           );
         }
-      }
+      },
+
+      {
+        args: [ new Error('forth') ],
+        level: 'error',
+        verify: function(entry) {
+          assert.ok(entry.data.startsWith('Error: forth'));
+        }
+      },
+
+      {
+        args: [ 'fifth message', new Error('fifth') ],
+        level: 'error',
+        verify: function(entry) {
+          assert.ok(entry.data.startsWith('fifth message: Error: fifth'));
+        }
+      },
     ];
 
     it('should properly write log entries', function(done) {
-      async.each(testData, function(test, callback) {
```

Comment:
This seems to be the only reference to `async` so I think it should be removed from the top and `devDependencies`.